### DISCUSSION TITLE
Custom overlay for more playground equipment types?

### DISCUSSION THREAD

**Discussion by Strubbl** [Helium314/SCEE#590] [Ideas]
I really love the SC things overlay and that i can map benches, trees, waste bins and playground equipment in the same overlay while being at a playground. But the overlay is missing some playground equipment types (compare https://wiki.openstreetmap.org/wiki/DE:StreetComplete/Playground with https://wiki.openstreetmap.org/wiki/DE:Key:playground - sorry for the DE links only).

What would make more sense to include playground equipment types for mapping? To create a custom overlay in SCEE? Is this even possible? 
Or would it be better to improve the situation in SC? 
Or any other idea?

**Comment by mnalis** [comment]
So, those in  https://wiki.openstreetmap.org/wiki/DE:StreetComplete/Playground/NotAvailable are missing? 

I could be wrong, but AFAIK those tag lists are defined in another project: id-tagging-schema (used not only by SC and SCEE, but also by iD web browser editor, EveryDoor, Vespucci etc), probably around here: https://github.com/openstreetmap/id-tagging-schema/tree/main/data/presets/playground

So doing it there would improve situation for everyone (those definitions are relatively simple JSON text files, so one might be able to suggest PR just by looking at several similar features and doing some copy/paste/replace)

> Is this even possible?

I'm not sure how hard would it be for SCEE to have additional list of POIs in addition to regular id-tagging-schema. Separate overlay just for playground equipment  might work, but is not ideal IMHO.

**Reply by Strubbl** [reply]
> So, those in https://wiki.openstreetmap.org/wiki/DE:StreetComplete/Playground/NotAvailable are missing?

Yes. I often see hammocks, spring boards, spinner and spinning discs, which i cannot map. 

> I could be wrong, but AFAIK those tag lists are defined in another project: id-tagging-schema (used not only by SC and SCEE, but also by iD web browser editor, EveryDoor, Vespucci etc), probably around here: https://github.com/openstreetmap/id-tagging-schema/tree/main/data/presets/playground

This sounds like a good idea. I need to research this. It would be the best solution to have it for all editors!

**Reply by mcliquid** [reply]
Since I have already added a change to the id-tagging-scheme **once**, I wanted to point out that it can be very time-consuming with the icons. But I just realised that most of them have a generic icon anyway 😃

**Reply by Helium314** [reply]
> I'm not sure how hard would it be for SCEE to have additional list of POIs in addition to regular id-tagging-schema

Currently SCEE only supports adding POIs that are in id-tagging-schema, and changing this might be quite a bit of work (possibly even requiring to fork [osmfeatures](https://github.com/westnordost/osmfeatures)).

Adding the features to id-tagging-schema is likely the best solution.
